docs/05-internals/design/principles/triplestore-updates.md

```diff
@@ -36,10 +36,7 @@ the same data, and may eventually provide functionality to facilitate
 this coordination.)
 
 We can assume that each SPARQL update operation will run in its own
-database transaction with an isolation level of 'read committed'. This
-is what GraphDB does when it receives a SPARQL update over HTTP (see
-[GraphDB SE
-Transactions](http://graphdb.ontotext.com/documentation/standard/storage.html#transaction-control)).
+database transaction with an isolation level of 'read committed'.
```

Comment:
Is this still true?

Reply:
It seems to be still valid statement if you read the whole paragraph.